src/tools/profile/read_profile.c

```diff
@@ -584,7 +814,7 @@ int parse_args(int argc, char **argv, options_t *opts)
     }
 
     if (optind >= argc) {
-        printf("Error: missing profile file argument\n");
+        printf("Error: missing profile file argument\n\n");
```

Comment:
`fprintf(stderr, ...);` ?

btw, maybe add `print_error` macro to not forget to include `"Error"` message:
```
#define print_error(_err_msg_fmt, ...) \
    fprintf(stderr, "Error"_err_msg_fmt, ##__VA_ARGS__)
```